<!-- -->
<launch>

<arg name="waypoint_file" default="$(find configurations)/config/waypoints.txt" />
<arg name="retryCount" default="10" />
<arg name="angularSpeedRadPS" default="0.8" />
<arg name="linearSpeedMPS" default="0.8" />
<arg name="pubRate" default="7" />
<arg name="destinationTolerance" default="0.4" />
<arg name="mode" default="1" />


  
  <node pkg="simple_nav" type="simple_nav_node" name="simple_nav_node" output="screen">
    <param name="waypoint_file" value="$(arg waypoint_file)" />
     <param name="mode" value="$(arg mode)" />
    <param name="retryCount" value="$(arg retryCount)" />
    <param name="angularSpeedRadPS" value="$(arg angularSpeedRadPS)" />
    <param name="linearSpeedMPS" value="$(arg linearSpeedMPS)" />
    <param name="pubRate" value="$(arg pubRate)" />
    <param name="destinationTolerance" value="$(arg destinationTolerance)" />
  </node>
  
</launch>
